### PR TITLE
2020 privacy japanese translation

### PR DESCRIPTION
**Pull request by ksakae1216**
Please review for me.
Part of #586

### CODE REVIEW

File: src/content/ja/2020/privacy.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+        <td class="numeric">8%</td>
+      </tr>
+      <tr>
+        <td>google.com</td>
+        <td><code>NID</code></td>
+        <td class="numeric">6%</td>
+      </tr>
+      <tr>
+        <td>doubleclick.net</td>
+        <td>unknown</td>
```
Comment (bazzadp) [inline src/content/ja/2020/privacy.md:143]: Can we translate all instanced of "unknown"?

**Review by bazzadp — CHANGES_REQUESTED** [review]
Looks good! Thanks!

One minor thing I spotted.

File: src/content/ja/2020/privacy.md (hunk 1 above)
Comment (ksakae1216) [reply src/content/ja/2020/privacy.md:143]: @bazzadp Yes, We Japanese understand "unknown".

Comment (bazzadp) [reply src/content/ja/2020/privacy.md:143]: OK so just leave it as is and merge?

Comment (ksakae1216) [reply src/content/ja/2020/privacy.md:143]: No problem. 👍

Comment (bazzadp) [reply src/content/ja/2020/privacy.md:143]: Sorry bit confused. So that is a "Yes" to merging it as is?

Comment (ksakae1216) [reply src/content/ja/2020/privacy.md:143]: Yes, please merge.

**Pull request merged**